app/models/notification.rb

```diff
@@ -279,71 +279,6 @@ def self.prioritized_list(user, count: 30, types: [])
     notifications.to_a
   end
 
-  # TODO(osama): deprecate this method when redesigned_user_menu_enabled is removed
```

Comment:
I think this comment is out-of-date. We're still using this `recent_report` method for the legacy hamburger menu

https://github.com/discourse/discourse/blob/83f1a13374bc3906919744f48071511c12912df9/app/controllers/notifications_controller.rb#L33-L34